DRAFT.md

```diff
@@ -682,8 +681,7 @@ explicit constexpr indirect(allocator_arg_t, const Allocator& alloc);
 
 7. _Mandates_: `T` is a complete type.
 
-8. _Effects_: Constructs an `indirect` owning a default constructed `T` and
-  stores the address in `p`. `alloc` is direct-non-list-initialized with `alloc`.
+8. _Effects_: TODO.
```

Comment:
`std::vector` has a constructor taking an allocator (but without the `std::allocator_arg_t` parameter: https://eel.is/c++draft/vector#cons

```
constexpr explicit vector(const Allocator&) noexcept;
[1] Effects: Constructs an empty vector, using the specified allocator.
[2] Complexity: Constant.
```

Reply:
"using the specified allocator" is the phrase-of-power that we need. Great spot!